### PR TITLE
add case for app

### PR DESCRIPTION
**Pull request by Zero-Rock**
## What this PR does / why we need it:

add case for app

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/auto_test/tests/dop/app/app-manage.spec.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      await app.createApp(formData);
+      await page.click(`text=${formData.name}`);
+      await page.waitForNavigation();
+      expect(page.url()).toMatch(/\d+\/repo$/);
+      await page.click('text=Settings');
+      await app.editApp();
+      await app.deleteApp(formData.name);
+      await page.close();
+    });
+    test('mobile app manage', async ({ page }) => {
```
Comment (daskyrk) [inline shell/auto_test/tests/dop/app/app-manage.spec.ts:42]: Missing another two app types.

File: shell/auto_test/tests/dop/app/notify.spec.ts [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      );
+      await wait(1);
+      await expectExist(`td:has-text("${name}")`, 1);
+      await notify.deleteRecord(name);
+    });
+    test('notify-group-member', async ({ page, wait, expectExist }) => {
+      const notify = new Notify(page);
+      await notify.openNewGroupModal();
+      const name = `member-${Date.now()}`;
+      notifyGroupNames.push(name);
```
Comment (daskyrk) [inline shell/auto_test/tests/dop/app/notify.spec.ts:65]: The api will not throw error even member not exist ?

File: shell/auto_test/tests/dop/project-manage.spec.ts [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Role('Manager', () => {
+  test.describe('project manage', () => {
+    test.beforeEach(async ({ page, goTo }) => {
+      await goTo('orgCreateProject');
+      await page.waitForEvent('requestfinished');
+    });
+    test.afterEach(async ({ page }) => {
+      await page.close();
+    });
+    test('msp project', async ({ page, goTo }) => {
```
Comment (daskyrk) [inline shell/auto_test/tests/dop/project-manage.spec.ts:33]: Missing dop project type.

File: shell/auto_test/tests/pages/base.ts [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+  async fillData<T extends {}>(key: string, data: T) {
+    await this.page.click(`#${key}`);
+    await this.page.fill(`#${key}`, data?.[key] ?? '');
+  }
+
+  /**
+   * @description 上传文件
+   * @param filePath {string} 文件相对于 erda-ui/shell/ 的路径，
+   * @param selector {string} input 选择器
```
Comment (daskyrk) [inline shell/auto_test/tests/pages/base.ts:40]: Use english comment.

File: shell/auto_test/tests/pages/notify.ts [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+        await this.page.click('text=send test notification');
+        break;
+      case 'external api':
+        await this.fillData('targets', data);
+        break;
+      case 'external user':
+        await this.clickButton('add external user');
+        await this.page.fill('.ant-modal >> .ant-table >> :nth-match(input[type="text"], 1)', 'erda-external-user');
+        await this.page.fill('.ant-modal >> .ant-table >> :nth-match(input[type="text"], 2)', 'erda@terminus.io');
+        await this.page.fill('.ant-modal >> .ant-table >> :nth-match(input[type="text"], 3)', '13100000000');
```
Comment (daskyrk) [inline shell/auto_test/tests/pages/notify.ts:76]: Could use label text instead of position?

File: shell/auto_test/tests/pages/notify.ts [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+        break;
+      case 'member role':
+        await this.page.click('.ant-select-selection-overflow');
+        await this.page.click('.ant-modal >> text=应用主管');
```
Comment (daskyrk) [inline shell/auto_test/tests/pages/notify.ts:80]: Should test in English mode, so there should not exist any Chinese word.

Comment (Zero-Rock) [reply shell/auto_test/tests/pages/notify.ts:80]: this api for querying the role list currently doesn't support internationalization，and have feedback with back-end development

**Pull request merged**